python/pyproject.toml

```diff
@@ -48,6 +47,7 @@ Changelog = "https://github.com/sift-stack/sift/tree/main/python/CHANGELOG.md"
 [project.optional-dependencies]
 development = [
   "grpcio-testing==1.13",
+  "npTDMS~=1.9",
```

Comment:
I think it'd be better to put this into a more specific `tdms` group, like I did with ssl related stuff [here](https://github.com/sift-stack/sift/blob/main/python/pyproject.toml#L63), so people can specify `sift-stack-py[tdms]` in their requirements if they need this specifically.